### PR TITLE
use sbt-pekko-build addPekkoModuleDependency

### PR DESCRIPTION
**Pull request by pjfanning**
https://github.com/pjfanning/sbt-pekko-build supports addPekkoModuleDependency which has tooling that supports testing with Pekko snaphots.

### CODE REVIEW

**Comment by pjfanning** [comment]
the link validator issue is an intermittent failure - not associated with this change

**Review by samueleresca — APPROVED** [review]

File: project/Dependencies.scala [hunk 1]
```diff
@@ -11,25 +11,18 @@ import Versions._
...
 
 object Dependencies {
-  lazy val benchmarkDependencies = Def.setting(Seq(
+  lazy val benchmarkDependencies = Seq(
     "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
     "io.dropwizard.metrics" % "metrics-core" % "4.2.28",
     "ch.qos.logback" % "logback-classic" % logbackVersion,
     "org.slf4j" % "log4j-over-slf4j" % slf4jVersion,
     "org.testcontainers" % "kafka" % testcontainersVersion % IntegrationTest,
-    "org.apache.pekko" %% "pekko-slf4j" % pekkoVersion % IntegrationTest,
```
Comment (raboof) [inline project/Dependencies.scala:20]: Looks like we could remove `Versions.pekkoVersion` now too

**Review by raboof — APPROVED** [review]

File: project/Dependencies.scala (hunk 1 above)
Comment (pjfanning) [reply project/Dependencies.scala:20]: It's still used in ParadoxSettings.scala. I can move the val there.

Comment (raboof) [reply project/Dependencies.scala:20]: yeah seems fine to just use `PekkoCoreDependency.version` directly there

**Pull request merged**